app/views/helpers/languages.php

```diff
@@ -487,7 +487,7 @@ public function getLevelsLabels($index = null)
     {
         if (!isset($__languagesLevels)) {
             $__languagesLevels = array(
-                -1 => __('Unspecified', true),
+                -1 => __p('level', 'Unspecified', true),
```

Comment:
I’m adding the translatable string `Unspecified` while it already exists somewhere else. So this will allow translators to translate the string `Unspecified` as a language level and `Unspecified` in the list dropdown separately. The first parameter `level` is a context indicator that gets displayed in Transifex (this is explained on the [wiki](http://en.wiki.tatoeba.org/articles/show/interface-translation)).